liiatools_pipeline/ops/cans_org.py

```diff
@@ -0,0 +1,98 @@
+import os
+from importlib import resources
+
+import pandas as pd
+from dagster import In, Out, get_dagster_logger, op
+from fs.base import FS
+from ruamel.yaml import YAML
+
+from liiatools.cans_pipeline.summary_sheet_mapping import add_summary_sheet_columns
+from liiatools.common import pipeline as pl
+from liiatools.common.constants import SessionNamesCANSMapping
+from liiatools.common.data import DataContainer
+from liiatools.common.pipeline import open_file
+from liiatools_pipeline.assets.common import shared_folder, workspace_folder
+from liiatools_pipeline.util.utility import opendir_location
+
+yaml = YAML()
+
+log = get_dagster_logger()
+
+
+@op(
+    out={
+        "session_folder": Out(FS),
+    }
+)
+def create_cans_session_folder() -> FS:
+    session_folder, session_id = pl.create_session_folder(
+        workspace_folder(), SessionNamesCANSMapping
+    )
+    """
+    Create CANS session folder and move required files for mapping into it
+    """
+    log.info("Creating CANS session folder")
+    session_folder = session_folder.opendir(SessionNamesCANSMapping.INCOMING_FOLDER)
+
+    current_folder = opendir_location(workspace_folder(), "current/cans/PAN")
+    pl.move_files_for_sharing(
+        current_folder,
+        session_folder,
+    )
+
+    return session_folder
+
+
+@op(
+    ins={
+        "session_folder": In(FS),
+    },
+)
+def cans_summary_sheet_mapping(
+    session_folder: FS,
+):
+    """
+    Add CANS Summary sheet mapping columns to CANS files
+    and export enriched files to shared folder
+    """
+    files = session_folder.listdir("/")
+    log.info(f"Files in session folder: {files}")
+
+    with resources.files("liiatools").joinpath(
+        "cans_pipeline", "spec", "summary_sheet_mapping.yml"
+    ).open("r") as f:
+        mapping = yaml.load(f)
+
+    with resources.files("liiatools").joinpath(
```

Comment:
I would add this functionality to the liiatools\cans_pipeline\spec\__init__.py file along with the load_schema and load_pipeline_config. Can just be load_summary_column_order and load_summary_sheet_mapping. This will save you having to rewrite this code for the unit tests you've written too